benchmarks/transport_montecarlo_numba_formal_integral_p.py

```diff
@@ -8,36 +8,62 @@
 import tardis.transport.montecarlo.formal_integral as formal_integral
 from benchmarks.benchmark_base import BenchmarkBase
 from tardis import constants as c
+from tardis import run_tardis
+from tardis.io.configuration.config_reader import Configuration
 from tardis.model.geometry.radial1d import NumbaRadial1DGeometry
-from tardis.util.base import intensity_black_body
-
 
+# Error in all functions: Terminating: fork() called from a process already using GNU OpenMP, this is unsafe. 
+@skip_benchmark
 class BenchmarkMontecarloMontecarloNumbaNumbaFormalIntegral(BenchmarkBase):
     """
     Class to benchmark the numba formal integral function.
     """
 
+    def __init__(self):
+        super().__init__()
+        self.config = None
+        filename = "data/tardis_configv1_benchmark.yml"
+        path = self.get_relative_path(filename)
+        self.config = Configuration.from_yaml(path)
+        self.Simulation = run_tardis(
```

Comment:
If this function is intended to benchmark formal integral only, there should be the a fixture for this TARDIS benchmark simulation object.

Reply:
Ignore above comments, this part is in __init__ not being timed